zk/stages/stage_sequence_execute.go

```diff
@@ -47,7 +47,12 @@ func SpawnSequencingStage(
 	}
 
 	if lastBatch < highestBatchInDs {
-		return resequence(s, u, ctx, cfg, historyCfg, lastBatch, highestBatchInDs)
+		if err = cfg.dataStreamServer.UnwindToBatchStart(lastBatch + 1); err != nil {
+			return err
+		}
```

Comment:
Is this going to work in resequence mode? We retrieve the data to be resequenced from ds. If ds is unwound, we won't have anything to resequence.

Reply:
We should flag activate the ability to resequence from the ds? This behaviour is needed when l1 recovering - are both scenarios valid simultaneously?

Reply:
Yes, we should read data from ds when resequencing, but only one mode (from `resequence`, `l1recover`, `limbo`, `sequence`) can be activated at one time.
Looks like resequencing is broken in the latest CI: https://github.com/0xPolygonHermez/cdk-erigon/actions/runs/12427988658/job/34698775817#step:4:190, but somehow the CI job didn't expose the error.